### ISSUE TITLE
Fix false negatives for double slash comments in no-empty-comment

### ISSUE THREAD

**Issue by evilebottnawi** [stylelint/stylelint#4037]
> Clearly describe the bug

`comment-no-empty` doesn't report on SCSS empty comment

> Which rule, if any, is the bug related to?

`comment-no-empty`

> What CSS is needed to reproduce the bug?

e.g.

```scss
//

a {
  width: 00.10em;
}
```

> What stylelint configuration is needed to reproduce the bug?

e.g.

```json
{
  "rules": {
    "comment-no-empty: true
  }
}
```

> Which version of stylelint are you using?

e.g. `10.0.1`

> How are you running stylelint: CLI, PostCSS plugin, Node.js API?

CLI

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

SCSS

> What did you expect to happen?

warning

> What actually happened (e.g. what warnings or errors did you get)?

No warning

**Comment by evilebottnawi** [comment]
@jeddy3 looks it is expected https://github.com/stylelint/stylelint/blob/master/lib/rules/comment-no-empty/index.js#L23 :confused: 

I think it was done due:
```scss
// Here example:
//
// a {
//   color:red;
// }
```
So we have one `//` empty comment. 

Solution: i think when we check `//` comment we need check comment before and after also, and report only when **all** comments are empty.

Warning:
```scss
//
```
```scss
//
//
//
```

Not warning:
```scss
//
// Comment
//
```

```scss
// Somethings
//
//
// Something
```

It

**Comment by vankop** [comment]
create issue in `stylelint-scss` https://github.com/kristerkari/stylelint-scss/issues/361

**Comment by evilebottnawi** [comment]
Let's close issue here

**Comment by kristerkari** [comment]
stylelint-scss v3.17.0 includes `scss/comment-no-empty` rule:
https://github.com/kristerkari/stylelint-scss/releases/tag/3.17.0
